frontend-instructor/src/context/authContext.jsx

```
@@ -9,8 +9,8 @@ const useAuth = () => {
 
 const LoadingSpinner = () => {
   return (
-    <div className="flex items-center justify-center h-screen">
-      <div className="animate-spin rounded-full h-10 w-10 border-t-4 border-blue-500"></div>
+    <div className='flex items-center justify-center h-screen'>
+      <div className='animate-spin rounded-full h-10 w-10 border-t-4 border-blue-500'></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)